taxonomies/categories.txt

```diff
@@ -72073,13 +72089,6 @@ lt:Salotos su daržovėmis
 nl:Groentesalade
 pl:Sałatki z warzywami
 
-<en:Plant-based foods
```

Comment:
Why is this removed?

Reply:
Because this category corresponds to nothing that exists, there are no "vegetable pasta". It could correspond to two things:
* Durum wheat pasta colored with vegetable. 
A category already exist for those products [Flavored and colored dry pastas](https://world.openfoodfacts.org/category/flavored-and-colored-dry-pastas). We could eventually create one for fresh pasta which contain higher amount of vegetable (up to 30%).
* Alternatives to wheat pasta made from chickpeas, split peas, lentils. They all have a category already too.